api/src/test/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExpiryControllerTest.java

```diff
@@ -563,7 +564,7 @@ public void handleInitialCreditsExpiry_singleAlert_forExhaustedAndByoBilling() t
     workspace = workspaceDao.findDbWorkspaceByWorkspaceId(workspace.getWorkspaceId());
     workspaceDao.save(
         workspace
-            .setBillingAccountName("billingAccounts/byo-account")
+            .setBillingAccountName(fullBillingAccountName("byo-account"))
```

Comment:
this is the main change.  Use a common method instead of manually constructing a string